### ISSUE TITLE
Fix duplicated properties when using Shader LODs

### ISSUE THREAD

**Issue by diogovtx** [AmplifyCreations/AmplifyShaderEditor-Feedback#141]
![image](https://user-images.githubusercontent.com/7001068/199619906-bd0f1af8-d8ff-4505-a508-bc2d4edc4a97.png)

![image](https://user-images.githubusercontent.com/7001068/199619929-49e6936b-d87d-42f9-a0e9-b20535808897.png)

![image](https://user-images.githubusercontent.com/7001068/199619988-fd5e4635-30c9-49c9-9fff-97d53804ca54.png)

![image](https://user-images.githubusercontent.com/7001068/199620025-cba56ff9-bdf6-433e-a2de-71ecde4871a2.png)

**Comment by Artie-Reedmand** [comment]
It is also problem with CBUFFER. Right now when you create shader with different lods (subshaders) CBUFFER adjusted independently on each subshader. It leads to 2 problems when you switch active subshader from highest default to another:
Different order in CBUFFER parameters leads to bugs in rendering or completely brokes rendering of object with this shader
If CBUFFER is somehow different at subshaders (not only order, but parameter's count too), it leads to broken SRP Batching of this shader

As a result, it will be great to have ability to automatically "copy" CBUFFER from highest level subshader to all another. In other words, so that all subshaders have CBUFFER parameters identical to highest subshader

**Comment by Dawie3565** [comment]
@diogovtx 
please bump this to as soon as possible as it's been open too long with bug tag 
cheers

**Comment by diogovtx** [comment]
Been going through multiple fixes to address the CBUFFER (#98, #658, etc) in v1.9.9.13 and the property duplication issues in earlier versions.

I believe this issue is now stale. If that isn't the case, please reply with a repro shader and I'll reopen, assuming I manage to verify the issue still persists.

**Comment by Dawie3565** [comment]
@Artie-Reedmand yes lets verify soon as ase v1.9.9.13 is out released and if found again open new ticket or verified closed 

just let me know in discord or chat we had before 

Thank you
